metal/metrics.py

```diff
@@ -160,6 +144,33 @@ def f1_score(gold, pred, **kwargs):
     return fbeta_score(gold, pred, beta=1.0, **kwargs)
 
 
+def roc_auc_score(gold, probs, ignore_in_gold=[], ignore_in_pred=[]):
+    """Compute the ROC AUC score, given the gold labels and predicted probs.
+
```

Comment:
Would put a check here to make sure only used for `k=2` (or below in metric_score)

Reply:
See definition in `sklearn`- is actually defined